script.js

```diff
@@ -161,19 +162,11 @@ function QuickWiki() {
               var dy = d.offset().top;
               var xgap = e.pageX - dx;
               var ygap = e.pageY - dy;
-              if (!$.browser.msie) {
-                e.preventDefault();
```

Comment:
The code `e.preventDefault()` was written in `title.mousedown` function, which is anyway a `<div id="wiki-title">` content, having no links as such. So I dont think default clicking action was even needed here. Functionally, nothing is affected till now